server/storage/players.lua

```diff
@@ -340,6 +340,46 @@ local function removePlayerFromGang(citizenid, group)
     removeFromGroup(citizenid, GroupType.GANG, group)
 end
 
+---@param citizenid string
+---@param job PlayerJob
+---@return boolean success if operation is successful.
+local function setPlayerPrimaryJob(citizenid, job)
+    local affectedRows = MySQL.update.await('UPDATE players SET job = ? WHERE citizenid = ?', {json.encode(job), citizenid})
+    return affectedRows > 0
```

Comment:
should we be asserting that the operation is successful, or do you think it's better for it to fail silently?